experimenter/tests/integration/nimbus/conftest.py

```diff
@@ -98,17 +98,13 @@ def firefox_options(firefox_options):
 def selenium(selenium, experiment_name, kinto_client, base_url, slugify):
     yield selenium
 
-    from nimbus.pages.experimenter.summary import SummaryPage
-
-    experiment_slug = str(slugify(experiment_name))
-    try:
-        summary = SummaryPage(selenium, urljoin(base_url, experiment_slug)).open()
-        summary.end_and_approve()
-        kinto_client.approve()
-        summary = SummaryPage(selenium, urljoin(base_url, experiment_slug)).open()
-        summary.wait_for_complete_status()
-    except Exception:
-        pass
+    if os.getenv("CIRCLECI") is None:
+        experiment_slug = str(slugify(experiment_name))
+        try:
+            helpers.approve_and_launch_experiment(experiment_slug)
+            kinto_client.approve()
```

Comment:
Wait why is this launching it?  I thought the helper here was to end experiments that are already live?